package.json

```diff
@@ -219,7 +219,9 @@
     "storybook-build": "build-storybook -c .storybook -o docs-ui/.storybook-out --quiet",
     "webpack-profile": "NO_TS_FORK=1 yarn -s webpack --profile --json > stats.json",
     "install-api-docs": "cd api-docs && yarn install",
-    "build-derefed-docs": "yarn install-api-docs && node api-docs/index.js api-docs/openapi.json",
+    "build-spectacular-docs": "OPENAPIGENERATE=1 sentry django spectacular --file tests/apidocs/openapi-spectacular.json --format openapi-json --validate --fail-on-warn",
+    "build-derefed-docs": "yarn build-deprecated-docs && yarn build-spectacular-docs && node api-docs/index.js tests/apidocs/openapi-spectacular.json tests/apidocs/openapi-derefed.json" ,
+    "build-deprecated-docs": "yarn install-api-docs && node api-docs/index.js api-docs/openapi.json tests/apidocs/openapi-deprecated.json",
```

Comment:
this seems like an unusual place to invoke python, I think if we want to have those shortcuts they should be in the `Makefile`